pages/home/configure/engine.mdx

```diff
@@ -8,30 +8,39 @@ description: "Arcade Engine Configuration"
 Arcade Engine's configuration is a [YAML file](https://yaml.org/) with the following sections:
```

Comment:
In this file:
- Sorted config sections alphabetically
- Added missing sections: `cache, security, storage`